stable/drupal/templates/drupal-hostpath.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and .Values.persistence.enabled .Values.persistence.drupal.hostPath (not .Values.persistence.drupal.ExistingClaim) -}}
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: v1
+    kind: PersistentVolume
+    metadata:
+      name: {{ template "fullname" . }}-hostpath
+    spec:
+      accessModes:
+        - {{ .Values.persistence.drupal.accessMode | quote }}
+      capacity:
+        storage: {{ .Values.persistence.drupal.size | quote }}
+      hostPath:
+        path: {{ .Values.persistence.drupal.hostPath | quote }}
+  - apiVersion: v1
```

Comment:
couldn't you reuse the `drupal-pvc.yaml` and template that based on the `hostPath` value?